src/zeit/content/article/breaking.py

```diff
@@ -1,4 +1,4 @@
-from zeit.content.article.edit.interfaces import IBreakingNewsBody
```

Comment:
Das ist glaub ich übers Ziel hinausgeschossen: Die Funktionalität ist hier, dass man im "Eilmeldung anlegen" Formular simpel schnell was in ein Textfeld reinschreibt, und das ordentlich in den Body des neu angelegten Artikels als Paragraph geschrieben wird.

Dass das vorher _außerdem auch noch_ auf den Banner-"Artikel" angewandt wurde, ist... einer der Gründe, dass wir diesen Unsinn jetzt endlich umbauen.

Ich glaub man muss hier eigentlich nur das `article_id` entfernen (und `banner_matches` geeignet anpassen), und das war's dann.